.htaccess

```diff
@@ -57,6 +57,9 @@ RewriteRule ^__version__$ version.json [L]
 # heartbeat.cgi returns 200 if the DB and memcached are both working, and 500 otherwise.
 RewriteRule ^__heartbeat__$ heartbeat.cgi [L]
 
+# New single page interface for filing bugs
+RewriteRule ^new-bug$ new_bug.cgi [L]
```

Comment:
I think this needs to be [L,QSA]:
https://httpd.apache.org/docs/current/rewrite/flags.html#flag_qsa